database/localdbg/.env

```diff
@@ -0,0 +1,3 @@
+GOOSE_DRIVER=postgres
```

Comment:
these dababase local files can be added in second PR later.

Reply:
agree